tex/chapters/literature.tex

```diff
@@ -1,11 +1,106 @@
-even specific equations in each chapter, this is an
-equation~(\ref{equ:random_data}).
+- Explain Prisoner's Dilemma
 
-\lipsum
+- Axelrod's original tournament
+
+- Work to reproduce Axelrod's tournament ~\cite{Knight2016}
+
+\begin{tabular}{c c c c}\label{tab:tournament_refs}
+Year & Reference & Number of Strategies & Type\\
+\hline
+1979 & & 13 & Standard\\
+1979 & & 64 & Standard\\
+1984 & & 64 & Evolutionary\\
+1991 & & 13 & Noisy\\
+2005 & & 223 & Varied\\
+2012 & & 13 & Standard\\
+\hline
+% \caption{An overview of published tournaments}
+\end{tabular}
+
+is often used to model systems in biology ~\cite{Sigmund1999}, sociology ~\cite{Franken2005},
+psychology ~\cite{Ishibuchi2005}, and economics ~\cite{Chong2005}.
+
+
+\section{Fingerprinting}\label{sec:fingerprinting}
+
+\begin{definition}\label{def:joss-ann}
+If $A$ is a strategy for playing the iterated prisoner's dilemma, then the \textbf{Joss-Anne of A}, $JA(A, x, y)$ is a strategy which has a probablility $x$ of choosing move $C$, a probability $y$ of chosing move $D$, and otherwise uses the response appropriate to strategy $A$.
+\end{definition}
+
+The notation $JA$ comes from the initials of the names Joss and Anne.
+Joss was a strategy submitted to one of Axelrod’s original tournaments and it would occasionally defect without provocation in the hopes of a slight improvement in score.
+Anne is the first name of A. Stanley who suggested the addition of random cooperation (refs from ashlock paper) instead of random defection.
+When $x + y = 1$, the original strategy is not used, and the resulting behavior is a random strategy with probabilities $(x, y)$.
+In more general terms, a $JA$ strategy is an alteration of a strategy $A$ that causes the strategy to be played with random noise inserted into the responses.
+
+\begin{definition}\label{def:fingerprint}
+A \textbf{Fingerprint} $F_A(S, x, y)$ with $0 \leq x, y \leq 1$, $x+y \leq 1$ for strategy $S$ and probe $A$, is the function that returns the expected score of strategy $S$ against $JA(A, x, y)$ for each possible $(x, y)$.
+\end{definition}
+
+\begin{definition}\label{def:double-fingerprint}
+The \textbf{Double Fingerprint} $F_{AB}(S, x, y)$ with $0 \leq x, y \leq 1$ returns the expected score of strategy $S$ against $JA(A, x, y)$ if $x+y \leq 1$, and $JA(B, 1-y, 1-x)$ if $x+y \geq 1$.
+\end{definition}
+
+\begin{definition}\label{def:dual}
+Strategy $A'$ is said to be the \textbf{Dual} of strategy $A$ if $A$ and $A'$ can be written as finite-state machines that are identical except that their responses are reversed.
+\end{definition}
+
+\begin{theorem}\label{fingerprint-unit-square}
+If $A$ and $A'$ are dual strategies, then $F_{AA'}(S, x, y)$ is identical to the function $F_A(S, x, y)$ extened over the unit square.
+\end{theorem}
+
+
+\section{Example Fingerprint Construction}
+
+There are several steps to constructing the Fingerprint of a strategy a basic familiarity of Markov Chains is required.
```

Comment:
this is the analytical fingerprint right. Perhaps worth including a discussion of how that's what's concentrated on in the literature. But thanks to your algorithm it can be extended to any strategy. Could be a second little section (or a later chapter) - but should still be signposted.